js/ui/windowManager.js

```diff
@@ -468,34 +468,22 @@ var WindowManager = class WindowManager {
 
         this.settings = new Settings({schema_id: 'org.cinnamon.muffin'});
 
-        const settingsState = {
-            'desktop-effects-on-dialogs': global.settings.get_boolean('desktop-effects-on-dialogs'),
-            'desktop-effects-on-menus': global.settings.get_boolean('desktop-effects-on-menus'),
-            'show-snap-osd': global.settings.get_boolean('show-snap-osd'),
-            'show-tile-hud': global.settings.get_boolean('show-tile-hud'),
-            'workspace-osd-visible': global.settings.get_boolean('workspace-osd-visible'),
-            'workspace-osd-x': global.settings.get_int('workspace-osd-x'),
-            'workspace-osd-y': global.settings.get_int('workspace-osd-y'),
-            'workspace-osd-duration': global.settings.get_int('workspace-osd-duration'),
-            'alttab-switcher-style': global.settings.get_string('alttab-switcher-style'),
-            'desktop-effects': this.settings.get_boolean('desktop-effects'),
-            'workspaces-only-on-primary': this.settings.get_boolean('workspaces-only-on-primary'),
-            'snap-modifier': this.settings.get_string('snap-modifier'),
-        };
+        this.desktop_effects_enabled = this.settings.get_boolean('desktop-effects');
```

Comment:
https://jsperf.com/dot-notation-vs-square-bracket-notation

Bracket notation is faster in Spidermonkey and V8. I think this part of the PR should be reverted.